docs/Project.toml

```diff
@@ -1,5 +1,7 @@
 [deps]
+CFTime = "179af706-886a-5703-950a-314cd64e0468"
 CairoMakie = "13f3f980-e62b-5c42-98c6-ff1f3baf88f0"
+ClimaOcean = "0376089a-ecfe-4b0e-a64f-9c555d74d754"
```

Comment:
we're supposed to dev ClimaOcean right?

Reply:
yes, that shouldn't be there... if I committed it, it was by accident